launcher/ui/MainWindow.cpp

```diff
@@ -1560,26 +1560,27 @@ void MainWindow::setCatBackground(bool enabled)
         QDateTime now = QDateTime::currentDateTime();
         QDateTime birthday(QDate(now.date().year(), 11, 30), QTime(0, 0));
         QDateTime xmas(QDate(now.date().year(), 12, 25), QTime(0, 0));
-        QString cat;
+        QString cat = APPLICATION->settings()->get("BackgroundCat").toString();
+
+        qDebug() << "The choosen cat is " << cat << ".";
```

Comment:
```suggestion
        qDebug() << "The choosen cat is" << cat;
```
Or just remove it altogether maybe